nexus/tests/integration_tests/silos.rs

```diff
@@ -1667,34 +1652,50 @@ async fn test_silo_user_views(cptestctx: &ControlPlaneTestContext) {
     );
 }
 
+/// Create a user in a SamlJit Silo for testing
+///
+/// For local-only Silos, use the real API (via `create_local_user()`).
```

Comment:
Good question!

With `LocalOnly`, the Oxide system is the source of truth of user data, so we need a way for people to manage that data.  That happens through the CRUD API.  With `SamlJit`, the IdP is the source of truth, and the Oxide system's user data gets updated when people authenticate with SAML.  The rack's data is essentially a read-only cache that gets updated from the IdP during authentication.  The assumption in this case is that the user doesn't _want_ to manage user data with an API -- in fact, they explicitly want to ensure that the rack's user data cannot be modified independently of their IdP.

In the limit I could see adding an API here to remove a previously-JIT-created user (or group).  And maybe a privileged way to JIT-create-or-update a user (or group) (although some security teams might not like this, since it allows the system to diverge from the IdP).  This would sort of amount to the same thing as a CRUD API.  But the story is pretty different: it's not "here, Oxide rack, let me tell you about the users and groups I want" -- it's more like a few knobs to pre-fill or dump parts of a cache.